src/ArrayToXml.php

```diff
@@ -26,7 +26,9 @@ public function __construct(
         array $domProperties = [],
         ?bool $xmlStandalone = null
     ) {
-        $this->document = new DOMDocument($xmlVersion, $xmlEncoding);
+        $this->document = $xmlEncoding === null ?
```

Comment:
This seems like a functional change. Could you revert all changes to this file?

Reply:
Reverted it. Sorry, I'm running PHP 8.1...just checked, it works on PHP 8.0.